packages/wonder-blocks-button/src/themes/default.ts

```diff
@@ -1,101 +1,289 @@
 import * as tokens from "@khanacademy/wonder-blocks-tokens";
 
+const {semanticColor} = tokens;
+
 // The underline-offset is the distance between the text baseline and the
 // bottom of the underline. This is necessary to prevent the underline from
 // breaking with descenders.
 const textUnderlineOffset = tokens.spacing.xxxSmall_4;
 
 const theme = {
     color: {
-        bg: {
-            /**
-             * Color
-             */
-            // color="default"
-            action: {
-                default: tokens.color.blue,
-                active: tokens.color.activeBlue,
-                inverse: tokens.color.fadedBlue,
-            },
-            // color="destructive"
-            critical: {
-                default: tokens.color.red,
-                active: tokens.color.activeRed,
-                inverse: tokens.color.fadedRed,
+        /**
+         * Primary
+         */
+        filled: {
+            // kind=primary / color=default / light=false
+            progressive: {
+                // filled
+                ...semanticColor.action.filled.progressive,
+                disabled: {
```

Comment:
Would it be helpful to have the `disabled` object as part of the semantic color tokens? (I might have asked this when we were pairing but I can't remember 😅)